CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ Main (unreleased)
 
 - Introduce ebpf exporter v2 integration. (@tpaschalis)
 
+- Add ability to configure agent to report to grafana.com the usage of feature flags (@marctc)
+
+
 ### Enhancements
 
 - integrations-next: Integrations using autoscrape will now autoscrape metrics
```

Comment:
```suggestion
- integrations-next: Integrations using autoscrape now auto-scrape metrics.
```

Reply:
This wasn't introduce by me but we can change anyway

Reply:
The sentence goes on in the line below, maybe this change should be reverted?